ibm/ibm-how-to/launch-ubuntu-instances.rst

```diff
@@ -0,0 +1,153 @@
+#. Choose the desired profile for your VSI under the `Profile` section.
+#. Select an existing ssh key or create a new one by pressing on :guilabel:`Add SSH key` and following the instructions.
+#. Customize any other settings as needed.
+#. Press on the blue :guilabel:`Create virtual server` button to launch your VSI.
+
+Via IBM Cloud CLI
+-----------------
+
+**Prerequisites**
+
+- IBM Cloud account
+- IBM Cloud CLI installed on Ubuntu
+
+
+**Step 1: Log in to IBM Cloud**
+
+1. After installing the CLI, log in to your IBM Cloud account using the command:
+
+.. code-block:: bash
+
+   ibmcloud login
+
+2. If you have a federated ID, use:
+
+.. code-block:: bash
+
+   ibmcloud login --sso
+
+**Step 2: Target a Resource Group**
+
+1. List available resource groups:
+
+.. code-block:: bash
+
+   ibmcloud resource groups
+
+2. Target the desired resource group:
+
+.. code-block:: bash
+
+   ibmcloud target -g <Resource_Group_Name>
+
+**Step 3: Create SSH Key**
+
+1. Generate an SSH key for VSI access if you don't have one:
+
+.. code-block:: bash
+
+   ssh-keygen -t rsa -b 2048
+
+2. Add the SSH key to IBM Cloud:
+
+.. code-block:: bash
+
+   ibmcloud is key-create MyKey --public-key ~/.ssh/id_rsa.pub
+
+Replace "MyKey" with a unique name for your key.
+
+**Step 4: Gather Information for Your VSI**
+
+Before launching your Virtual Server Instance (VSI), you need to collect the VPC ID, Zone, Profile Name, and Image ID. Here's how to retrieve these details:
+
+1. **List VPCs to Get VPC ID:**
+
+First, you need the ID of the Virtual Private Cloud (VPC) where your instance will reside.
+
+.. code-block:: bash
+
+   ibmcloud is vpcs
+
+This command lists all VPCs in your account. Identify the VPC you wish to use and note its ID.
+
+2. **Identify the Availability Zone:**
+
+Your VSI needs to be hosted in an availability zone. To list available zones in your region:
+
+.. code-block:: bash
+
+   ibmcloud is regions
+
+Choose a region, then list the zones in that region:
+
+.. code-block:: bash
+
+   ibmcloud is zones <Region-Name>
+
+Replace `<Region-Name>` with your chosen region. Select a zone and note its name.
+
+3. **Choose an Instance Profile:**
+
+Instance profiles determine the VSI's virtual CPU and memory characteristics.
+
+.. code-block:: bash
+
+   ibmcloud is instance-profiles
+
+Choose a profile suitable for your workload and note the profile name.
+
+4. **Select an Image for Your VSI:**
+
+Images define the operating system and software configuration of your VSI.
+
+.. code-block:: bash
+
+   ibmcloud is images
+
+Choose an image that meets your requirements (e.g., Ubuntu) and note the Image ID.
+
+**Step 5: Launch the Virtual Server Instance**
+
+With all necessary information in hand, create your VSI:
+
+.. code-block:: bash
+
+   ibmcloud is instance-create MyInstance <VPC-ID> <Zone> <Profile-Name> <Image-ID> --keys <Key-ID>
```

Comment:
Consider encouraging the user to export each value from each step into an environment variable so they could run something like `ibmcloud is instance-create MyInstance "$VPC_ID" "$ZONE" "$PROFILE_NAME" "$IMG_ID" --keys "$KEY_ID"`.

Just an idea, because managing all these values manually in the cloud console is mildly annoying.

Reply:
after discussing with several parties, have decided this is far too hand holdy. need to make this more of a how-to less of a tutorial.